hack/check-everything.sh

```diff
@@ -27,6 +27,19 @@ kb_root_dir=$tmp_root/kubebuilder
 # Run verification scripts.
 ${hack_dir}/verify.sh
 
+# HACK
+k8s_repo_url=https://github.com/kubernetes/kubernetes.git
+k8s_clone_dir=$tmp_root/kubernetes
+
+echo "Cloning Kube repository from $k8s_repo_url..."
+git clone $k8s_repo_url $k8s_clone_dir
+
+cd $k8s_clone_dir
+echo "Building Kube from source code..."
+make
+echo "Kube built successfully."
```

Comment:
I just want to make sure it can clone and compile. Later on I will simply copy the built binary into the expected directory.